src/System.Management.Automation/namespaces/FileSystemContentStream.cs

```diff
@@ -222,50 +225,6 @@ internal class FileSystemContentReaderWriter : IContentReader, IContentWriter
             _suppressNewline = suppressNewline;
         }
 
-        /// <summary>
-        /// Constructor for the content stream
-        /// </summary>
-        /// <param name="path">
-        /// The path to the file to get the content from.
-        /// </param>
-        /// <param name="mode">
-        /// The file mode to open the file with.
-        /// </param>
-        /// <param name="access">
-        /// The file access requested in the file.
-        /// </param>
-        ///  <param name="share">
-        ///    The file share to open the file with
-        ///  </param>
-        /// <param name="delimiter">
-        /// The delimiter to use when reading strings. Each time read is called, all contents up to an including
-        /// the delimiter is read.
-        /// </param>
-        /// <param name="encoding">
-        /// The encoding of the file to be read or written.
-        /// </param>
-        /// <param name="waitForChanges">
-        /// If true, we will perform blocking reads on the file, waiting for new content to be appended
-        /// </param>
-        /// <param name="provider">
-        /// The CmdletProvider invoking this stream
-        /// </param>
-        /// <param name="isRawStream">
-        /// Indicates raw stream.
-        /// </param>
-        public FileSystemContentReaderWriter(
```

Comment:
Why was this removed?

Reply:
It is unneeded and used __internal__ constructor and we can safely remove it. (Otherwise we have to fix it to follow our changes in another constructor with delimiter parameter.)